### ISSUE TITLE
Can not reimplement the result with the same settings

### ISSUE THREAD

**Issue by ZrrSkywalker** [Strawberry-Eat-Mango/PCT_Pytorch#20]
Hi thanks so much for your excellent reimplementation, but I got the accuracy of 92.9% on modelnet40.
My training settings are as follows, and I think it's the same as yours:

lr 0.0001,   batch_size 32,   epoch 250,   useSGD,   using I GPU
Accuracy: 92.9%

**Comment by SleepySheepShip** [comment]
Same with you, < 93.0%

**Comment by Yi-Shi94** [comment]
Are you guys using the aligned version or the unaligned version of ModelNet40？

**Comment by xiaohe3625** [comment]
Using GPU : 0 from 1 devices
Test :: test acc: 0.932739, test avg acc: 0.899901

**Comment by Stronger-Huang** [comment]
> Using GPU : 0 from 1 devices Test :: test acc: 0.932739, test avg acc: 0.899901
do you just inference with the checkpoint 'model.t7' the author gives?

**Comment by Stronger-Huang** [comment]
> Using GPU : 0 from 1 devices Test :: test acc: 0.932739, test avg acc: 0.899901

do you just inference with the checkpoint 'model.t7' the author gives?
